src/hooks/useCountries.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { getCountries } from '../services/country.js';
+
+export function useCountries() {
+  const [countries, setCountries] = useState([]);
+  const [type, setType] = useState(['Antartica']);
```

Comment:
you'd probably want it to default to an 'all countries' state rather than starting with a subset of all the countries

Reply:
also you'll only have one 'type' at a time so this should initialize to a string rather than an array with a string in it